app/(nms)/users/page.tsx

```diff
@@ -0,0 +1,100 @@
+"use client"
+
+import { useQuery } from "@tanstack/react-query"
+import { listUsers } from "@/utils/accountQueries"
+import { UserEntry } from "@/components/types"
+import Loader from "@/components/Loader"
+import { useAuth } from "@/utils/auth"
+import PageHeader from "@/components/PageHeader"
+import { Button, ContextualMenu, MainTable, Modal } from "@canonical/react-components"
+import PageContent from "@/components/PageContent"
+import { useState } from "react"
+import SyncOutlinedIcon from "@mui/icons-material/SyncOutlined";
+import { ChangePasswordModal, CreateUserModal, DeleteModal } from "./modals"
+
+type modalData = {
+  user: UserEntry
+  type: "delete" | "change password" | "create user"
+}
+
+export default function Users() {
+  const [modalData, setModalData] = useState<modalData | null>(null);
+  const auth = useAuth()
+  const query = useQuery<UserEntry[], Error>({
+    queryKey: ['users', auth.user?.authToken],
+    queryFn: () => listUsers({ authToken: auth.user ? auth.user.authToken : "" }),
+    enabled: auth.user ? true : false,
+    retry: (failureCount, error): boolean => {
+      if (error.message.includes("401")) {
+        return false
+      }
+      return true
+    },
+  })
+  if (query.status == "pending") { return <Loader text="loading..." /> }
+  if (query.status == "error") {
+    if (query.error.message.includes("401")) {
+      auth.logout()
+    }
+    return <p>{query.error.message}</p>
+  }
+  const users = Array.from(query.data ? query.data : [])
+  const tableContent = users.map((user) => {
+    return {
+      key: user.username,
+      columns: [
+        { content: user.username },
+        {
+          content: (
+            <ContextualMenu
+              links={[
+                {
+                  children: "Delete account",
+                  disabled: user.role == 1,
+                  onClick: () => setModalData({ user: user, type: "delete" })
+                }, {
+                  children: "Change password",
+                  onClick: () => setModalData({ user: user, type: "change password" })
+                }
+              ]}
+              hasToggleIcon
+              position={"right"}
+            />
+          ),
+        },
+      ],
+    };
+  });
+
+  return (
+    <>
+      <PageHeader title={`NMS Accounts (${users.length})`}>
```

Comment:
```suggestion
      <PageHeader title={`Users (${users.length})`}>
```

We are interchangibly using "accounts" and "users", let's make sure to always use the same word when we identify this concept.

Reply:
Updated